src/gds/ua_gds_ns.c

```diff
@@ -0,0 +1,1045 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ *    Copyright 2018 (c) Markus Karch, Fraunhofer IOSB
+ */
+
+
+#include "ua_registration_manager.h"
+#include "server/ua_server_internal.h"
+
+#ifdef UA_ENABLE_GDS
+
+
+#ifdef UA_ENABLE_GDS_CM
+#include "ua_certificate_manager.h"
+
+
```

Comment:
This file should be generated automatically, using https://github.com/OPCFoundation/UA-Nodeset/blob/master/GDS/Opc.Ua.Gds.NodeSet2.xml

@jpfr do you agree?

See also:
https://github.com/open62541/open62541/blob/6ad0222f619935347589f84eea08454ebcf22c0a/examples/nodeset/CMakeLists.txt#L36